packages/config/src/lib/createConfig.ts

```diff
@@ -96,6 +97,7 @@ export const createConfig = ({
   nodeModulesDirectories = [],
   stripAllPfStyles = false,
   blockLegacyChrome,
+  devtool,
```

Comment:
Can we do `devtool = false` to follow other configs with default values?

Reply:
yeah sure thing, just sent this up